graylog2-server/src/main/java/org/graylog2/alerts/types/FieldValueAlertCondition.java

```diff
@@ -147,16 +135,27 @@ protected CheckResult runCheck() {
                 case LOWER:
                     triggered = result < threshold.doubleValue();
                     break;
+                default:
+                    triggered = false;
             }
 
             if (triggered) {
+                final List<MessageSummary> summaries = Lists.newArrayList();
```

Comment:
We know `fieldStatsResult.getSearchHits()`, so we can initialize the list with the correct size.